src/test_l1_plat_power.c

```diff
@@ -152,8 +152,10 @@ void test_l1_plat_power_negative_PLAT_INIT (void)
     UT_ASSERT_EQUAL_FATAL(ret, PWRMGR_SUCCESS); // Ensure the returned value is PWRMGR_SUCCESS
 
     // Variation/Step 02: Call PLAT_INIT() - again open interface
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
please review this model, and upgrade your switches to support KVP so we can at runtime control the error codes

-> https://github.com/rdkcentral/ut-core/discussions/55

Reply:
also ensure we don't use fatal anywhere, we need to double-check if we're using version 2+ of ut-core.